lib/block/transaction.go

```diff
@@ -180,6 +180,15 @@ func (bt *BlockTransaction) SaveBlockOperations(st *storage.LevelDBBackend) (err
 		return errors.FailedToSaveBlockOperaton
 	}
 
+	if bt.blockHeight < 1 {
```

Comment:
It feels like patching after the fact. How come we call `SaveBlockTransaction` with a default-initialized height ?

Reply:
@Geod24 It's a mistake this part is missed.

Reply:
@spikeekips : Since we have `if bt.blockHeight < 1`, I assume it's not always missing. In which case is it ?

Reply:
@Geod24 `BlockTransaction.Height` is 0, means `BlockTransaction` is get from storage. `BlockTransaction.Height` is only available when it is saved.